.github/workflows/deploy.yml

```diff
@@ -0,0 +1,39 @@
+name: deploy
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: 'actions/checkout@v3'
+
+    - id: 'auth'
+      uses: 'google-github-actions/auth@v0'
+      with:
+        credentials_json: '${{ secrets.GCP_CREDENTIALS }}'
+
+    - name: 'Create GCP BigQuery Credential file'
+      run: touch /home/runner/work/rapidpro-api-client/rapidpro-api-client/big-query-credentials.json | base64
+
+    - name: 'Add Credential from Github Secrets'
+      env:
+        DATA: ${{ secrets.GOOGLE_SERVICE_CREDENTIALS }}
+      run: echo $DATA > /home/runner/work/rapidpro-api-client/rapidpro-api-client/big-query-credentials.json
+
+    - id: 'deploy'
+      uses: 'google-github-actions/deploy-cloud-functions@v0'
+      with:
+          name: unified_alerts
```

Comment:
Hey @vivek-kumar-pandey  Are we going to use the `unified_alerts` cloud function or will be creating a new one?

Reply:
Will be creating a new one. Fixing it.